pkg/controllers/nodeclaim/inplaceupdate/patch.go

```diff
@@ -82,6 +102,33 @@ func CalculateVMPatch(
 	return update
 }
 
+// Given AKS machine support PUT, but not PATCH, the AKS machine object will be patched directly, while the returning Machine object is just a tracker for logging purposes.
```

Comment:
Confusing comment -- it can't really be patched directly because there is no PATCH, right?

I think you mean "we will apply the patch locally and perform a full machine update w/ etag check" or something like that?